app/views/posts/index.html.erb

```diff
@@ -0,0 +1,28 @@
+<h1>Posts</h1>
+
+<table>
+  <thead>
+    <tr>
+      <th>Body</th>
+      <th colspan="3"></th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <% @posts.each do |post| %>
+      <tr>
+        <td><%= post.body %></td>
+        <% if user_signed_in?%>
+          <td><%= post.user.name %></td>
+        <% end %>
+        <% if user_signed_in? && post.user_id == current_user.id %>
+          <td><%= link_to 'Destroy', post, method: :delete, data: { confirm: 'Are you sure?' } %></td>
+        <% end %>
```

Comment:
That's what we called logic
Kindly move that logic into helpers.

Reply:
Could you explain a little bit how we are supposed to move this to helpers? 
I don't see any serious logic here. Just basic conditions to determine what gets displayed, No serious logic in my view.

Thanks